### PR TITLE
Hotels: Using UTC dates for search form

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
@tbergq Would be great to have a tests for it since you are fixing a bug here. Maybe you'll decide to use some library for it later but it should still work the same...

**Comment by tbergq** [comment]
Yes, you are right, I will add some test for it

**Comment by tbergq** [comment]
@mrtnzlml 🚀

File: packages/localization/src/DateFormatter.js [hunk 1]
```diff
@@ -46,7 +47,7 @@ function pad(number) {
  * formatter. Every function is localized and you should never call formatting
  * on the raw Date object.
  */
-function DateFormatter(rawDate: Date = new Date()) {
+function DateFormatter(rawDate: Date = DateUtils.getUTCNow()) {
```
Comment (mrtnzlml) [inline packages/localization/src/DateFormatter.js]: I still believe this method should not accept the `Date` type... :)

Comment (tbergq) [reply packages/localization/src/DateFormatter.js]: Yes, maybe for a later refactor.

**Pull request merged**